com.unity.cinemachine/Samples~/Sample Scenes/Shared/Scripts/SetAimTarget.cs

```diff
@@ -0,0 +1,28 @@
+using Cinemachine;
+using UnityEngine;
+
+public class SetAimTarget : MonoBehaviour
+{
+    [Tooltip("A transform to hold the position the camera is aiming at in world space")]
+    public Transform AimTarget;
+    
+    [Tooltip("Cinemachine camera that defines the aim target")]
+    public CinemachineVirtualCameraBase CmCamera;
```

Comment:
It's not great to be specifying a vcam here.  It will fail as soon as a second vcam is activated, so is not a strong solution.  Better would be to dynamically obtain the current active vcam and find its 3rdPersonAim, if any